Lib/fontmake/font_project.py

```diff
@@ -541,6 +540,18 @@ def save_otfs(
                 if "GSUB" in gsub_src:
                     font["GSUB"] = gsub_src["GSUB"]
 
+            # Read autohinting parameters from ufo lib if present
+            if (
+                "com.schriftgestaltung.customParameter.InstanceDescriptorAsGSInstance.TTFAutohint options"
+                in ufo.lib
+                and autohint is None
+            ):
+                autohint = ufo.lib[
+                    "com.schriftgestaltung.customParameter.InstanceDescriptorAsGSInstance.TTFAutohint options"
```

Comment:
let's keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Reply:
I don't understand the hint. Is this the same remark as Simon’s regarding re-using the `autohint` variable?

Reply:
yes, we don't want to repeat ourselves, so move the constant at the top, all good now